src/utils/currentDate.js

```diff
@@ -0,0 +1,7 @@
+const CurrentDate = () => {
```

Comment:
So this is a Javascript standard, but all variables and function name conventions need to be either in `camelCase` or `snake_case`. You have to pick one and stick with one. Classes, environment variables, and etc are the only exceptions where this would be allowed.